search-ui/src/sbc-common-components/components/MobileDeviceAlert.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-row justify="center">
+  <v-row justify="center v-overlay">
```

Comment:
did you mean to add this as a class? I think this will break when it tries to render?

Reply:
I don't think so as before this class, header was hidden behind the breadcrumb. I guess this is also how you implemented on ppr.

Reply:
oh interesting, okay sounds good